mrp_production_request/wizards/mrp_production_request_create_mo.py

```diff
@@ -0,0 +1,144 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Eficent Business and IT Consulting Services S.L.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from openerp import api, fields, models
+import openerp.addons.decimal_precision as dp
+
+
+class MrpProductionRequestCreateMo(models.TransientModel):
+    _name = "mrp.production.request.create.mo"
+    _description = "Wizard to create Manufacturing Orders"
+
+    @api.multi
+    def compute_product_line_ids(self):
+        self.product_line_ids.unlink()
+        res = self._prepare_lines()
+        product_lines = res[0]
+        # workcenter_lines = res[1]  # TODO: expand with workcenter_lines
```

Comment:
I prefer to keep it to not forget about this in future improvements, or to give some hint to anyone interested to implement this feature.